packages/node_modules/@node-red/nodes/core/function/10-function.html

```diff
@@ -464,6 +470,26 @@
                 }
             });
 
+            // 4294967295 is max in node.js timeout.
+            $( "#node-input-timeout" ).spinner({
+                min: 0,
+                max: 4294967294,
```

Comment:
```suggestion
            // 4294967 is max in node.js timeout.
            $( "#node-input-timeout" ).spinner({
                min: 0,
                max: 4294967,
```

Change to second resolution.